src/styles/_variables/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$color: #eee;
```

Comment:
Consider using more descriptive variable names. `$color` could be `$background-color`, `$size` could be `$font-size`, `$border` could be `$border-style`, `$padding` could be `$element-padding`, `$gap` could be `$margin`, `$days` could be `$weekdays`. 🎨